var/spack/repos/builtin/packages/m4/package.py

```diff
@@ -48,13 +48,11 @@ def configure_args(self):
         spec = self.spec
         args = ['--enable-c++']
 
-        # CFLAGS handling
-        cflags = copy.deepcopy(optflags[self.spec.compiler.name])
```

Comment:
the only thing you miss is:
```
import copy
```

Reply:
nevermind, i see that the whole setup is wrong as `optflags` is not defined either. In CP2K it's a local variable...